### ISSUE TITLE
how about image gradient loss's effect?

### ISSUE THREAD

**Issue by austingg** [wonbeomjang/mobile-hair-segmentation-pytorch#2]
hi, guys
How about the image gradient loss's effect?

**Comment by wonbeomjang** [comment]
I do not remember well it because I had done it a year ago. It was not much different from when I used the cross entropy function alone.
